lib/markdown-pdf.js

```diff
@@ -81,15 +89,20 @@ function htmlFromPreview(callback){
   //render markdown using Atom's Markdown-Preview package
   var cb = atom.clipboard;
   if(editorSelected()){
-    //save old clipboard contents
-    var old = cb.read();
-    //get html on clipboard
-    mdpreview.mainModule.copyHtml();
-
-    var html = cb.read();
-    //put old clipboard contents back
-    cb.write(old);
-    callback(html);
+    if(mdpreview.name == 'markdown-preview'){
+      //save old clipboard contents
+      var old = cb.read();
+      //get html on clipboard
+      mdpreview.mainModule.copyHtml();
+
+      var html = cb.read();
+      //put old clipboard contents back
+      cb.write(old);
+      callback(html);
+    } else {
+      // copy parsed markdown with maths scaled 200%
+      mdpreview.mainModule.copyHtml(callback, 200)
```

Comment:
Because our package renders maths asynchronously, I believe the best solution was to redefine `copyHtml` in our MPP to accept a callback rather than rely on the clipboard which would have to be polled. 

The second parameter of `200` is because when rendering svg elements (which is what our maths blocks are parsed to) appear to render at half the size compared to when you view the html in a browser window. `200` represents a 200% scaling of all the maths in the markdown to be converted to pdf.